desktop/app/lib/app-instance/index.js

```diff
@@ -22,6 +22,7 @@ AppInstance.prototype.anotherInstanceStarted = function () {
 
 AppInstance.prototype.isSingleInstance = function () {
 	if ( app.requestSingleInstanceLock() ) {
+		this.anotherInstanceStarted();
```

Comment:
`anotherInstanceStarted` calls `platform.restore`. Not sure if this works as a fix or if this will break other platforms (Mac, Linux).